.travis.yml

```diff
@@ -54,26 +54,26 @@ jobs:
     ############################
 
     # Fetch the linux archive
-    - travis_retry curl -L https://hydra.iohk.io/job/Cardano/cardano-wallet/cardano-wallet-shelley-linux64/latest/download/1 | tar xz
-    - find . -maxdepth 1 -type d -name "cardano-wallet-shelley-*" -exec mv \{} cardano-wallet-shelley-linux64 \;
-    - mv cardano-wallet-shelley-linux64/cardano-wallet-shelley cardano-wallet-shelley-linux64/cardano-wallet
-    - tar czf cardano-wallet-$TRAVIS_TAG-linux64.tar.gz cardano-wallet-shelley-linux64
-    - rm -rf cardano-wallet-shelley-linux64
+    - travis_retry curl -L https://hydra.iohk.io/job/Cardano/cardano-wallet/cardano-wallet-linux64/latest/download/1 | tar xz
+    - find . -maxdepth 1 -type d -name "cardano-wallet-*" -exec mv \{} cardano-wallet-linux64 \;
```

Comment:
This `cardano-wallet-*` will probably stop working though assuming the binaries from hydra will change

Reply:
Will deal with it when merged and the latest-hydra-link is updated.